### PR TITLE
Bugfixes from staging

### PR DESCRIPTION
**Pull request by AndrewTHEManeri**
These were all the changes and bugfixes we needed to apply for the 5.6 release.  We need to examine these and see which ones were stopgap and need new issues created.

### CODE REVIEW

**Comment by stella3d** [comment]
the fix for blink visuals i would consider stopgap : we need to refactor how we consume input.  we have an issue already for this : https://github.com/Unity-Technologies/EditorVR/issues/324

File: Tools/LocomotionTool/LocomotionTool.cs [hunk 1]
```diff
@@ -199,7 +199,12 @@ public void ProcessInput(ActionMapInput input, ConsumeControlDelegate consumeCon
 			this.SetUIBlockedForRayOrigin(rayOrigin, true);
 
 			if (DoTwoHandedScaling(consumeControl))
+			{
+				if (m_Preferences.blinkMode)
```
Comment (AndrewTHEManeri) [inline Tools/LocomotionTool/LocomotionTool.cs]: More permanent solution captured in #324

File: LICENSE [hunk 2]
```diff
@@ -1,11 +1,32 @@
...
-
-The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL UNITY TECHNOLOGIES APS OR ANY OF ITS AFFILIATES ("UNITY") BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-
-Except as contained in this notice, the "Unity" name/mark shall not be used in advertising or otherwise to promote the sale, use or other dealings in this Software without prior written authorization from Unity.
+ 
+Unity Companion License 1.0 (�License�)
```
Comment (mtschoen-unity) [inline LICENSE]: Fix unicode quotes?

File: Menus/ToolsMenu/ToolsMenuButton/ToolsMenuButton.cs [hunk 3]
```diff
@@ -519,6 +519,10 @@ void OnBackgroundButtonClick()
 
 		void OnSecondaryButtonClicked()
 		{
+			if (!implementsSecondaryButton)
+			{
+				return;
+			}
```
Comment (mtschoen-unity) [inline Menus/ToolsMenu/ToolsMenuButton/ToolsMenuButton.cs]: Lose brackets for single-line if

Comment (AndrewTHEManeri) [reply Menus/ToolsMenu/ToolsMenuButton/ToolsMenuButton.cs]: CSharp standards reference says brackets for single-line ifs are optional; neither required nor prohibited

File: LICENSE (hunk 2 above)
Comment (AndrewTHEManeri) [reply LICENSE]: good catch

File: Menus/ToolsMenu/ToolsMenuButton/ToolsMenuButton.cs (hunk 3 above)
Comment (mtschoen-unity) [reply Menus/ToolsMenu/ToolsMenuButton/ToolsMenuButton.cs]: Not required, but we've been doing this consistently throughout the codebase. The only exception is where an else has more than one line. Personally I'm in favor of consistency, and generally if you can eliminate lines of code, you can see more on screen. Especially in the case of an early-out, I'd say no brackets for sure.

**Pull request merged**